### PR TITLE
migrate pkg/proxy/ipvs/graceful_termination.go to structured logs

### PR DESCRIPTION
**Pull request by chymy**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
xref:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

**Comment by chymy** [comment]
/assign @andrewsykim
Thanks

File: pkg/proxy/ipvs/graceful_termination.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ func (q *graceTerminateRSList) add(rs *listItem) bool {
 		return false
 	}
 
-	klog.V(5).Infof("Adding rs %v to graceful delete rsList", rs)
+	klog.V(5).InfoS("Adding rs to graceful delete rsList", "rs", fmt.Sprintf("%v", rs))
```
Comment (aojea) [inline pkg/proxy/ipvs/graceful_termination.go:65]: rs doesn´t seems very meaninful, but I´m not familiar with ipvs enough to judge or give an alternative, @andrewsykim what do you think?

File: pkg/proxy/ipvs/graceful_termination.go [hunk 2]
```diff
@@ -150,13 +150,13 @@ func (m *GracefulTerminationManager) GracefulDeleteRS(vs *utilipvs.VirtualServer
...
 	}
-	klog.V(5).Infof("Adding an element to graceful delete rsList: %+v", ele)
+	klog.V(5).InfoS("Adding an element to graceful delete rsList", "element", fmt.Sprintf("%+v", ele))
 	m.rsList.add(ele)
 	return nil
 }
 
 func (m *GracefulTerminationManager) deleteRsFunc(rsToDelete *listItem) (bool, error) {
-	klog.V(5).Infof("Trying to delete rs: %s", rsToDelete.String())
+	klog.V(5).Infof("Trying to delete rs", "name", rsToDelete.String())
```
Comment (yangjunmyfm192085) [inline pkg/proxy/ipvs/graceful_termination.go:159]:
```suggestion
	klog.V(5).InfoS("Trying to delete rs", "name", rsToDelete.String())
```

File: pkg/proxy/ipvs/graceful_termination.go [hunk 3]
```diff
@@ -167,10 +167,10 @@ func (m *GracefulTerminationManager) deleteRsFunc(rsToDelete *listItem) (bool, e
 			//     (existing connections will be deleted on the next packet because sysctlExpireNoDestConn=1)
 			// For other protocols, don't delete until all connections have expired)
 			if utilipvs.IsRsGracefulTerminationNeeded(rsToDelete.VirtualServer.Protocol) && rs.ActiveConn+rs.InactiveConn != 0 {
-				klog.V(5).Infof("Not deleting, RS %v: %v ActiveConn, %v InactiveConn", rsToDelete.String(), rs.ActiveConn, rs.InactiveConn)
+				klog.V(5).Infof("Not deleting rs", "name", rsToDelete.String(), "ActiveConn", rs.ActiveConn, "InactiveConn", rs.InactiveConn)
```
Comment (yangjunmyfm192085) [inline pkg/proxy/ipvs/graceful_termination.go:170]:
```suggestion
				klog.V(5).InfoS("Not deleting rs", "name", rsToDelete.String(), "ActiveConn", rs.ActiveConn, "InactiveConn", rs.InactiveConn)
```

File: pkg/proxy/ipvs/graceful_termination.go [hunk 4]
```diff
@@ -167,10 +167,10 @@ func (m *GracefulTerminationManager) deleteRsFunc(rsToDelete *listItem) (bool, e
...
 				return false, nil
 			}
-			klog.V(5).Infof("Deleting rs: %s", rsToDelete.String())
+			klog.V(5).Infof("Deleting rs", "name", rsToDelete.String())
```
Comment (yangjunmyfm192085) [inline pkg/proxy/ipvs/graceful_termination.go:173]:
```suggestion
			klog.V(5).InfoS("Deleting rs", "name", rsToDelete.String())
```

**Pull request closed without merging**